README.md

```diff
@@ -34,9 +34,14 @@ Here are the download links for latest release:
     <td><a href="https://github.com/bblanchon/pdfium-binaries/releases/latest/download/pdfium-linux-v8.tgz">pdfium-linux-v8.tgz</a> (37 MB)</td>
   </tr>
   <tr>
-    <td>macOS</td>
-    <td><a href="https://github.com/bblanchon/pdfium-binaries/releases/latest/download/pdfium-darwin.tgz">pdfium-darwin.tgz</a> (6 MB)</td>
-    <td><a href="https://github.com/bblanchon/pdfium-binaries/releases/latest/download/pdfium-darwin-v8.tgz">pdfium-darwin-v8.tgz</a> (42 MB)</td>
+    <td>macOS (Intel)</td>
+    <td><a href="https://github.com/bblanchon/pdfium-binaries/releases/latest/download/pdfium-darwin-x64.tgz">pdfium-darwin-x64.tgz</a> (6 MB)</td>
+    <td><a href="https://github.com/bblanchon/pdfium-binaries/releases/latest/download/pdfium-darwin-x64-v8.tgz">pdfium-darwin-x64-v8.tgz</a> (42 MB)</td>
```

Comment:
Same here, since the build times-out, we should remove the link (see Windows build)